code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -779,3 +779,22 @@
 	if(prob(30))
 		M << "You should sit down and take a rest..."
 	..()
+
+/datum/reagent/toxin/delayed
+	name = "Delayed Toxin"
+	id = "delayed_toxin"
+	description = "Causes heavy toxin damage after a brief time of inactivity."
+	reagent_state = LIQUID
+	metabolization_rate = 0 //stays in the system until active.
+	toxpwr = 0
+	var/actual_toxpwr = 6
+	var/delay = 30
+
+/datum/reagent/toxin/delayed/on_mob_life(mob/living/M)
+	if(current_cycle > delay)
+		holder.remove_reagent(id, metabolization_rate * M.metabolism_efficiency)
+		M.adjustToxLoss(actual_toxpwr*REM, 0)
+		if(prob(10))
+			M.Weaken(1, 0)
+		. = 1
+	..()
```

Comment:
Because every single "delayed" poison actually does something every cycle and just scales up